kentikapi/client.go

```diff
@@ -132,25 +122,34 @@ func (c *Config) FillDefaults() {
 		c.APIURL = APIURLUS
 	}
 
-	if c.CloudExportHostPort == "" {
-		c.CloudExportHostPort = cloudExportHostPort
-	}
-
-	if c.SyntheticsHostPort == "" {
-		c.SyntheticsHostPort = syntheticsHostPort
-	}
-
 	if c.Timeout == nil {
 		c.Timeout = pointer.ToDuration(defaultTimeout)
 	}
 
 	c.RetryCfg.FillDefaults()
 }
 
-func makeConnForGRPC(c Config, hostPort string) (grpc.ClientConnInterface, error) {
+func parseKentikURL(apiURL *string) (string, bool) {
```

Comment:
```suggestion
func makeGRPCHostPort(apiURL string) (grpcHostPort string, tlsEnabled bool) {
```
1. This function does more than just parsing given URL.
2. No need for pointer to string
3. Named return parameters are great for making signature readable in such cases.

Reply:
Partially done; new comments below